omod/src/main/resources/apps/patient_dashboard_app.json

```diff
@@ -322,5 +322,29 @@
         }
       }
     ]
+  },
+  {
+    "id": "ugandaemr.dashboardwidget.ReturnDateToFacilityFromPharmacy",
+    "instanceOf": "coreapps.template.dashboardWidget",
+    "description": "Return Date to facility from Pharmacy",
+    "order": 2,
+    "config": {
+      "widget": "latestobsforconceptlist",
+      "icon": "icon-user-md",
+      "label": "RETURN DATE FROM PHARMACY TO FACILITY",
+      "concepts": "f6c456f7-1ab4-4b4d-a3b4-e7417c81002a"
+    },
+    "requiredPrivilege": "App: ugandaemr.widget.returnDateToFacility",
```

Comment:
Import the role and add the permission to the Role **HIV Clinic**

Reply:
@slubwama on second thought  I think this widget should have same privilege as next appointment widget